src/types/TransactionRecord.ts

```diff
@@ -0,0 +1,34 @@
+import {receiptToSdk, TransactionReceipt} from "./TransactionReceipt";
+import {ContractFunctionResult, contractFunctionResultToSdk} from "./ContractFunctionResult";
+import {TransferList, transferListToSdk} from "./TransferList";
+import {TransactionRecord as ProtoTransactionRecord} from "../generated/TransactionRecord_pb";
+import {timestampToDate} from "../util";
+import {TransactionId, transactionIdToSdk} from "./TransactionId";
+
+export type TransactionRecord = {
+    receipt: TransactionReceipt | null;
+    transactionHash: Uint8Array | string;
+    consensusTimestamp: Date;
+    transactionId: TransactionId;
+    memo: string;
+    transactionFee: number;
+    contractCallResult: ContractFunctionResult | null;
+    contractCreateResult: ContractFunctionResult | null;
+    transferList: TransferList | null;
```

Comment:
`TransferList` is an unnecessary indirection here; just use `AccountAmount[]` directly.

Reply:
I'd just call this field `transfers` at that point, actually.

Reply:
And maybe inline the `AccountAmount` definition into this file if it's not used anywhere else.

Reply:
@abonander The reasoning behind using `transferList` as the name and `TransferList` type is to make it as similar to the protobufs as possible. I was thinking if one were to look at our types and then at the protobufs it would be apparent what is what simply by looking at the names/types. Do you you think we should ignore this and instead make it as simple as we can?